core/domain/takeout_service_test.py

```diff
@@ -1053,6 +1181,7 @@ def test_export_data_for_full_user_nontrivial_is_correct(self):
             'last_started_state_translation_tutorial': self.GENERIC_EPOCH,
             'last_logged_in': self.GENERIC_EPOCH,
             'last_edited_an_exploration': self.GENERIC_EPOCH,
+            'last_created_an_exploration': self.GENERIC_EPOCH,
```

Comment:
Possibly we should be adding _msec here and elsewhere too. However, if this is not straightforward, feel free to file an issue for it (that enumerates the relevant places it should be added).

Reply:
I'm going to file an issue for this just for the sake of the PR, but I'll also address that PR (it should be a simple fix).

Reply:
Thanks, can you link the issue number here please?

Reply:
I decided to just resolve this here. Done.